src/firejail/profile.c

```diff
@@ -423,7 +423,7 @@ int profile_check_line(char *ptr, int lineno, const char *fname) {
 		return 0;
 	}
 	else if (strcmp(ptr, "noautopulse") == 0) {
```

Comment:
Does this need changing too?

Reply:
Nope, changing this would mean to hard-fail (with invalid/unknow-command) if a user has `noautopulse` in their .locals.

Reply:
Thanks for clearing things up. Looks ready for merge!